libraries/conf_helpers.rb

```diff
@@ -0,0 +1,66 @@
+module CernerSplunk
+  # Mixin Helper methods for the Splunk Config resource
+  module ConfHelpers
+    def stringify_config(config)
+      config.map do |section, props|
+        [section.to_s, props.map { |k, v| [k.to_s, v.to_s] }.to_h]
+      end.to_h
+    end
+
+    def read_config(conf_path)
+      return {} unless Pathname.new(conf_path).exist?
```

Comment:
minor, could store a reference to the Pathname.new here and reuse it below on line 16.